src/qt/transactionrecord.h

```diff
@@ -23,6 +24,8 @@ struct WalletTxStatus;
 class TransactionStatus
 {
 public:
+    static constexpr int TIME_AGO_OF_LAST_BLOCK_IN_SECONDS = 90 * 60;
```

Comment:
nit, this name is kind of weird to me.

Reply:
@promag 
Good naming is hard :)
Would you mind proposing a better name?